h/schemas/admin_group.py

```diff
@@ -35,6 +35,18 @@ def user_exists_validator(form, value):
     return user_exists_validator
 
 
+@colander.deferred
+def group_type_validator(node, kw):
+    group = kw.get('group')
+    if not group:
+        return colander.OneOf([key for key, title in VALID_GROUP_TYPES])
+
+    def validate(node, value):
+        if group.type != value:
+            raise colander.Invalid(node, _('Changing group type is currently not supported'))
```

Comment:
This isn't important but FWIW I personally don't really like the formal / technical-ness of phrases lile "currently not supported" for usability and how the app feels. "Changing group type isn't supported yet" sounds friendlier to me and is shorter, easier to parse, and the "yet" implies that it will be supported in the future. But this is something that I don't think we have any agreement or guidelines on - what is the language style / voice of Hypothesis? In the case of this admin form it doesn't matter though

Reply:
I'm inclined to agree. @dawariley may correct me but I don't think we have any written documentation on the voice of the UI (eg. how formal, what tense).